Documentation/ApiOverview/Backend/JavaScript/RequireJS/Dependency/Index.rst

```diff
@@ -6,6 +6,15 @@
 Dependency handling
 ===================
 
+.. deprecated::  12.0
+    The RequireJS project has been discontinued_ and was therefore
+    replaced by native ECMAScript v6/v11 modules in TYPO3 v12.0. The
+    infrastructure for configuration and loading of RequireJS
+    modules is deprecated with v12.0 and will be removed in TYPO3 v13. See
+    :ref:`RequireJS to ES6 migration <requirejs-migration>`.
+
+.. _discontinued: https://github.com/requirejs/requirejs/issues/1816
+
```

Comment:
Should be add a hint that the following code it outdated by now?

Reply:
Well the old code still works but causes deprecation warnings in v12

Reply:
with the attention box it should become clearer